src/envo/envo-edit.owl

```diff
@@ -40017,6 +40019,28 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105110> "manufactured plastisphere"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105110> <http://purl.obolibrary.org/obo/ENVO_06105106>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105111> (natural plastic)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105111> "A plastic which is composed primarily of polymers produced by processes which occur without human intervention."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105111> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_06105111> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105111> <http://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105111> "2021-05-07T09:09:37Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_06105111> "envoPlastics")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105111> "natural plastic"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105111> <http://purl.obolibrary.org/obo/ENVO_06105101>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105112> (synthetic plastic)
```

Comment:
Following-up on our side-conversation, we could add `manufactured plastic` as a synonym

Reply:
And please mark (e.g. with an editor note) that you'll add the corresponding subclass axiom in the second phase of the project